src/ex3/actions.js

```diff
@@ -0,0 +1,74 @@
+import { getPokemon, isListOfPokemons, isPokemon, readFile, writeToFile, pokemonExistsInTodoList, overwriteFile, pokemonToAscii } from './helper.js';
+import chalk from 'chalk';
+
+export async function get() {
+    const data = await readFile();
+    for (let i = 0; i < data.length; i++) {
+        console.log(chalk.yellow(data[i]))
+    }
+}
+
+export async function add(item) {
+
+    let response
+
+    if (isListOfPokemons(item)) {
+        const listOfPokemons = item.split(',');
+        response = chalk.green("Pokemons added successfully :)")
+        for (let i = 0; i < listOfPokemons.length; i++) {
+            let pokemon = await getPokemon(listOfPokemons[i])
+            if (pokemon === 0) {
+                console.log(chalk.red(`Wrong Pokemon number --> ${listOfPokemons[i]}`))
+                continue
+            }
+            pokemonToAscii(pokemon.data.sprites.front_default)
+            pokemon = pokemon.data.name
+            let doesPokemonExists = await pokemonExistsInTodoList(pokemon)
+            if (!doesPokemonExists) {
+                const res = await writeToFile(`catch ${pokemon}`)
+                if (res !== "Item has been added :)") {
```

Comment:
note that this line always returns false, I think because you return a "chalk" object and not really a string.